docs/source/converting_tensorflow_models.rst

```diff
@@ -47,12 +47,12 @@ Here is an example of the conversion process for a pre-trained ``BERT-Base Uncas
 
 .. code-block:: shell
 
-   export BERT_BASE_DIR=/path/to/bert/uncased_L-12_H-768_A-12
+    export BERT_BASE_DIR=/path/to/bert/uncased_L-12_H-768_A-12
 
-   transformers-cli convert --model_type bert \
-     --tf_checkpoint $BERT_BASE_DIR/bert_model.ckpt \
-     --config $BERT_BASE_DIR/bert_config.json \
-     --pytorch_dump_output $BERT_BASE_DIR/pytorch_model.bin
+    transformers-cli convert --model_type bert \
+      --tf_checkpoint $BERT_BASE_DIR/bert_model.ckpt \
+      --config $BERT_BASE_DIR/bert_config.json \
+      --pytorch_dump_output $BERT_BASE_DIR/pytorch_model.bin
```

Comment:
Just a note from the previous PR - this indentation here is absolutely not required by bash, it'd work w/o no indent at all.

(this is just an example, all is good otherwise)

Reply:
I just moved the whole block, making a special rule was too tiring ;-)

Reply:
got it - I was just showing this as an example that the indentation is not required in bash cl args ;) I personally never use it, but some people like to use it so it's easier to see where the command starts, so it may appear to others that it's a requirement.